driver/src/main/scala/api/MongoDriver.scala

```diff
@@ -2,26 +2,19 @@ package reactivemongo.api
 
 import java.util.concurrent.atomic.AtomicLong
 
-import scala.util.{ Try, Failure, Success }
```

Comment:
Please keep the import formatting, including the explicit imports rather than `_`